### PR TITLE
[bliss] bump version

### PR DESCRIPTION
**Pull request by matbesancon**
this updates bliss to the latest version 0.77.
I'm not sure yet the patches will still work

### CODE REVIEW

**Comment by matbesancon** [comment]
For some reason it seems not to detect GMP

**Comment by giordano** [comment]
You need to see _how_ the build system looks for GMP

**Comment by matbesancon** [comment]
Seems to be here:
```
  20   │ if(USE_GMP)
  21   │   find_path(GMP_INCLUDE_DIR NAMES gmp.h)
  22   │   find_library(GMP_LIBRARIES NAMES gmp libgmp mpir REQUIRED)
  23   │   if (MSVC)
  24   │     add_compile_options(/DBLISS_USE_GMP /I${GMP_INCLUDE_DIR})
  25   │   else()
  26   │     add_compile_options(-DBLISS_USE_GMP -I${GMP_INCLUDE_DIR})
  27   │   endif()
  28   │ endif(USE_GMP)
```

**Comment by giordano** [comment]
Does `-DGMP_INCLUDE_DIR=${includedir}` help?

File: B/bliss/build_tarballs.jl [hunk 1]
```diff
@@ -3,29 +3,27 @@
...
 if [[ "${target}" == *mingw* ]]; then
   atomic_patch -p1 ../patches/notimer.patch
 fi
-# build with GMP and store this information
-sed -i -e 's/^namespace bliss/#define BLISS_USE_GMP\n\nnamespace bliss/' defs.hh
-make -j${nproc} lib_gmp CFLAGS="-I$prefix/include -O3 -fPIC -I." LDFLAGS="$LDFLAGS -L$libdir" CC="$CXX"
-# there is no target for a shared library
-$CXX -shared -o libbliss.$dlext *.og $LDFLAGS -L$libdir -lgmp
+cd build
+cmake -DUSE_GMP=on -DGMP_INCLUDE_DIR=${includedir} ..
```
Comment (giordano) [inline B/bliss/build_tarballs.jl:22]: You're missing some [arguments to CMake](https://docs.binarybuilder.org/dev/build_tips/#CMake-builds)
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    ..
```

**Pull request closed without merging**